packages/app/src/components/Root/Root.tsx

```diff
@@ -101,10 +102,19 @@ const getMenuItem = (menuItem: ResolvedMenuItem, isNestedMenuItem = false) => {
   );
 };
 
+const GlobalHeader = ({ mountPoints }: { mountPoints: MountPoints }) =>
```

Comment:
Let us use the name application header here and global header in the plugin, that makes it later easier to find the responsible component.

```suggestion
const ApplicationHeader = ({ mountPoints }: { mountPoints: MountPoints }) =>
```